### ISSUE TITLE
Immediately saying "all renders finished" without rendering

### ISSUE THREAD

**Issue by captainrice** [f0e/blurGUI#1]
Am I missing dependencies? I use the original version of blur but I installed the GUI version with the provided installer and It won't render any of the videos I put through it, I just get the "all renders finished" immediately. Any help is appreciated, thank you!

**Comment by couleurm** [comment]
having issues with this too

**Comment by f0e** [comment]
yeah you're probably missing dependencies. i haven't bundled ffmpeg, vapoursynth and python (because i can't come up with a good way to do it) so you'll have to install them yourself for now

**Comment by captainrice** [comment]
As far as I'm aware, I have ffmpeg, vapoursynth and python installed. Not really sure what I'm doing wrong tbh.

**Comment by couleurm** [comment]
would be cool to have a way to redirect the vapoursynth/ffmpeg installation from blur to blurGUI

**Comment by MassivebutterpapperCodes** [comment]
> As far as I'm aware, I have ffmpeg, vapoursynth and python installed. Not really sure what I'm doing wrong tbh.

u forgot vapoursynth
